src/app/core/validators/signup-validators.ts

```diff
@@ -1,25 +1,31 @@
 import { AbstractControl, ValidationErrors } from '@angular/forms';
 
 export class SignupValidators {
+  /**
+   * @param control - The form control to validate
```

Comment:
É legal colocar uma descrição do que está acontecendo no tsdoc, pode parecer repetitivo com o returns as vezes, mas é mais claro desta maneira.

Algo nesse estilo:

```ts
  /**
   * Custom Validator to form-fields, check if the value has white space on it.
   * 
   * @param control - The form control to validate
   * @returns ValidationErrors if the passwords do not match; otherwise null
   */
```